cmd/empire/main.go

```diff
@@ -184,6 +186,18 @@ var EmpireFlags = []cli.Flag{
 		EnvVar: "EMPIRE_ECS_SERVICE_ROLE",
 	},
 	cli.StringFlag{
+		Name:   FlagECSLogDriver,
+		Value:  "json-file",
+		Usage:  "Log driver to use when running containers. Maps to the --log-driver docker cli arg",
+		EnvVar: "EMPIRE_ECS_LOG_DRIVER",
+	},
+	cli.StringSliceFlag{
+		Name:   FlagECSLogOpts,
+		Value:  &cli.StringSlice{},
+		Usage:  "Log driver to options. Maps to the --log-opt docker cli arg",
```

Comment:
`logopt` takes multiple arguments, which is exactly how it works with `docker run`:

```
empire server --ecs.logdriver syslog --ecs.logopt tag=brianz --ecs.logopt syslog-facility=daemon
```

...results in:

```
        "LogConfig": {
            "Type": "syslog",
            "Config": {
                "syslog-facility": "daemon",
                "tag": "brianz"
            }
        },
```